api/src/pcapi/static/backofficev3/js/addons/pc-form-field.js

```diff
@@ -5,14 +5,91 @@
  *
  */
 class PcFormField extends PcAddOn {
+  static ID = 'PCFormFieldId'
+  static PC_FORM_FIELD_FIELD_CLASS = 'pc-form-field-field'
+  static PC_FORM_FIELD_FIELD_EMPTY_SELECTOR = '.pc-form-field-empty-container'
+  static PC_REQUIRED_SELECTOR = '.pc-required'
 
   initialize = () => {
-    document.querySelectorAll(".pc-form-field-empty-container").forEach(($emptyFormField) => {
-      $emptyFormField.querySelectorAll(".value-element-form").forEach(($valueElementForm) => {
+    document.querySelectorAll(this.constructor.PC_FORM_FIELD_FIELD_EMPTY_SELECTOR).forEach(($emptyFormField) => {
```

Comment:
J'aurai rajouté un getter pour ça, ex:

```ts
get formFieldEmpty() {
  return document.querySelectorAll(this.constructor.PC_FORM_FIELD_FIELD_EMPTY_SELECTOR)
}
```

Puis :

```suggestion
    this.formFieldEmpty.forEach(($emptyFormField) => {
```